.drone.star

```diff
@@ -1959,8 +1959,12 @@ def makeGoGenerate(module):
 
 def notify(ctx):
     status = ["failure"]
+    channel = ""
     if ctx.build.event == "cron":
         status.append("success")
+        channel = "builds"
+    else:
+        channel = config["rocketchat"]["channel"]
```

Comment:
```suggestion
    channel = config["rocketchat"]["channel"]
    if ctx.build.event == "cron":
        status.append("success")
        channel = "builds"
```

Reply:
can we put `"builds"` also in a config variable?